tests/sort

```diff
@@ -6,7 +6,7 @@ pass=true
 
 for file in list inactive; do
     if ! diff --color=always -u <(grep -v -E '^(#|$)' "$file") <(grep -v -E '^(#|$)' "$file" | sort) > "$tempfile"; then
-        echo -e "\nFile \"$file\" not proerly sorted!\n\nDiff between current and the correct version:" 1>&2
+        echo -e "\nFile $file not proerly sorted!\n\nDiff between current and the correct version:" 1>&2
```

Comment:
The result/behavior seems to be changed as below. Maybe #110 is a misunderstanding?

```diff
- File "list" not proerly sorted!
+ File list not proerly sorted!
```

Reply:
The word `list` have never been present. but happy to add it

![image](https://github.com/user-attachments/assets/6b83e1ad-ec91-4a7d-a95c-f16c481f7ec5)

Reply:
Oh, I mean the output of the script, in the script, it's in the for loop variable.